### PR TITLE
feat(Card): Add missing card features

### PR DESCRIPTION
**Pull request by iamdavid0091**
1. Add static classes to all card components
2. Update styles
3. Add new card demos
4. Add support for vertical actions

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import styled, { css } from 'styled-components';
 import elevation, { elevationTransition } from '../mixins/elevation';
 import ripple from '../mixins/ripple';
 
+const ButtonComponent = props => (
+  <button className={`${props.className} card-action`}>{props.children}</button>
```
Comment (brad-decker) [inline src/components/Button.js]: A button component can exist outside of the card context so the static class name for button should not be card-action. It should instead be 'material-button' or something similar. smc-button (styled material components button?) if that were the adopted pattern we should also adopt smc-card

File: src/components/Card/Avatar.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import styled from 'styled-components';
 
-const Avatar = styled.div`
+const AvatarComponent = props => (
+  <div className={`${props.className} card-avatar`}>{props.children}</div>
```
Comment (brad-decker) [inline src/components/Card/Avatar.js]: For now this is fine but we may want to elevate the avatar outside of the card context to use in other components. At which time card-avatar would be refactored to smc(?)-avatar

File: src/components/Card/Card.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 import styled from 'styled-components';
 import elevation from '../../mixins/elevation';
 
-const Card = styled.div`
+const CardComponent = props => (
+  <div className={`${props.className} demo-card`}>{props.children}</div>
```
Comment (brad-decker) [inline src/components/Card/Card.js]: Not demo card, but just card or smc-card

**Comment by brad-decker** [comment]
@iamdavid0091 bumpity bump bump

File: src/components/Button.js (hunk 1 above)
Comment (iamdavid0091) [reply src/components/Button.js]: Done. Changed to `smc-card`.

File: src/components/Card/Avatar.js (hunk 2 above)
Comment (iamdavid0091) [reply src/components/Card/Avatar.js]: Done. Changed to `smc-avatar`.

Comment (iamdavid0091) [reply src/components/Card/Avatar.js]: Done. Changed to `smc-avatar`.

File: src/components/Card/Card.js (hunk 3 above)
Comment (iamdavid0091) [reply src/components/Card/Card.js]: Done. Changed to `smc-card`.

**Pull request merged**